app/assets/javascripts/angular/components/organizationSignatures/organizationSignatures.component.js

```diff
@@ -0,0 +1,128 @@
+            errorMessage: 'error.messages.surveys.loadQuestions',
+        });
+    };
+
+    const prepareData = data =>
+        data.map(signature => ({
+            first_name: signature.first_name,
+            last_name: signature.last_name,
+            updated_at: signature.updated_at,
+            code_of_conduct: signature.code_of_conduct,
+            statement_of_faith: signature.statement_of_faith,
+        }));
+
+    const buildExportCsvLink = searchText => {
+        const params = {
+            access_token: $http.defaults.headers.common.Authorization.slice(7),
+            include: '',
+            format: 'csv',
+            ...(searchText ? { 'filters[q]': searchText } : {}),
+        };
+
+        const query = Object.keys(params).reduce((accumulator, key, index) => {
+            return (
+                accumulator +
+                '&' +
+                encodeURIComponent(key) +
+                '=' +
+                encodeURIComponent(params[key])
+            );
+        }, '');
+
+        return (
+            envService.read('apiUrl') +
+            `/organizations/${this.orgId}/signatures?${query}`
+        );
+    };
+
+    const loadData = async (searchText, tableState) => {
+        this.isLoading = true;
+        this.signatures = [];
+        const start = (tableState && tableState.pagination.start) || 0;
+        const { data, meta } = await getSignatures(
+            this.orgId,
+            searchText,
+            start,
+        );
+        const numberOfPages = Math.ceil(meta.total / this.itemsPerPage);
+
+        this.signatureData = prepareData(data);
+        this.signatures = [...this.signatureData];
+        this.exportLink = buildExportCsvLink(searchText);
+
+        tableState.pagination.numberOfPages = numberOfPages;
+        tableState.pagination.currentPage =
+            Math.ceil(start / this.itemsPerPage) + 1;
+        tableState.pagination.pages = [];
+
+        for (let i = 0; i < numberOfPages; i++) {
+            tableState.pagination.pages.push(i);
+        }
+
+        tableState.pagination.totalItemCount = meta.total;
+        this.tableState = tableState;
+        this.isLoading = false;
+        $scope.$apply();
+    };
+
+    this.setPage = (page, searchText) => {
+        this.tableState.pagination.start = page * this.itemsPerPage;
+        loadData(searchText, this.tableState);
+    };
+
+    this.nextPage = searchText => {
+        this.tableState.pagination.start =
+            this.tableState.pagination.currentPage * this.itemsPerPage;
+        loadData(searchText, this.tableState);
+    };
+
+    this.previousPage = searchText => {
+        this.tableState.pagination.start =
+            (this.tableState.pagination.currentPage - 2) * this.itemsPerPage;
+        loadData(searchText, this.tableState);
+    };
+
+    this.load = tableState => {
+        tableState.pagination.numberOfPages = 0;
+        this.tableState = tableState;
+        loadData(_, tableState);
```

Comment:
`_` is probably undefined or the lodash object... Usually when you use `_` for a throwaway variable, you use it in an assignment. Am I missing something? Would one of these be better?
```suggestion
        loadData(undefined, tableState);
```
```suggestion
        loadData(null, tableState);
```